app/views/schedules/edit.html.erb

```diff
@@ -5,8 +5,9 @@
   <div class="col-md-8 col-md-offset-2">
     <%= form_for(@schedule) do |f| %>
         <%= render partial: 'fields', locals: {f: f, object: @schedule} %>
-        <%= button_tag 'Save', type: :submit, class: 'btn btn-sm btn-primary' %>
-        <%= link_to 'Cancel', schedules_path, class: 'btn btn-sm btn-primary btn-danger' %>
+
+        <%= button_to 'Save', edit_schedule_path(@schedule), class: 'btn btn-sm btn-primary' %>
```

Comment:
I don't think you need to change these lines for the #edit view. Once the user has been redirected to this view from #create, from then on the user should be able to change the Schedule, save, and carry on as before.